subprojects/kris-io/src/main/kotlin/ch/difty/kris/KRisIO.kt

```diff
@@ -64,6 +69,47 @@ public object KRisIO {
     @Throws(IOException::class)
     public fun process(inputStream: InputStream): List<RisRecord> = process(inputStream.bufferedReader())
 
+    /**
+     * Converts the RISFile lines provided by the reader into a stream of [RisRecord]s.
+     * May throw an [IOException] if the reader fails to deliver lines
+     * or a [KRisException] if the lines cannot be parsed successfully.
+     */
+    @JvmStatic
+    @Throws(IOException::class)
+    public fun processToStream(reader: Reader): Stream<RisRecord> = runBlocking(Dispatchers.IO) {
+        val lineFlow = BufferedReader(reader).lineSequence().asFlow()
+        Stream.builder<RisRecord>().also { recordsStream ->
```

Comment:
IMHO `also` semantically does not fit that well. I prefer to use `apply` in this case. Since I have a few other minor changes (small refactoring in the test, adding the new functionality to the guide, minor cleanup I noticed here but for stuff I introduced earlier), I will merge this PR as is and follow up with my own.

Reply:
Perfect, thanks! I'll try to keep your convention better next time

Reply:
You followed my conventions so well that you also copied my grammar mistakes from before :-) No, seriously, there's no complaint about you not following my conventions, as most changes in my follow up PR are fixing my own mistakes from before.

Only Regarding the scoping functions. To me the semantics of `apply` is about initialization or configurationof an object, whereas `also` uses the lambda to do something with the object after it has been used. To me `apply` is more fitting than `also` in our use case, as we prefill the builder before calling the build method. But I guess that's also opinionated.

Thanks for your contribution.